docs/resources.rst

```diff
@@ -10,3 +10,4 @@ Here are some resources and examples for using Otter-Grader:
   <https://econ140-spring2020.shinyapps.io/otter_quickstart/>`_
 * `Otter-Grader JupyterLab Extension <https://github.com/chrispyles/otter-grader-labextension>`_: A
   JupyterLab extension for authoring Otter Assign-formatted notebooks.
+* `Chris Pyles and Suraj Rampure presenting at Jupytercon 2023 <https://www.youtube.com/watch?v=9_x532_2T2w&list=PL_1BH3ug7n1Ih_Yy2TmM7MZ2zogSLZvzE&index=19>`_  along with a `Jupyterlite demo file <https://t.ly/bb17>`_
```

Comment:
Thanks for the PR, Eric! Would you mind using this URL for the Jupyterlite demo so that we don't need to rely on the short link? Thanks!

https://chrispyles.io/otter-grader-jupytercon-2023/lab/index.html

Reply:
Ok umm, I think i edited it, hope I got it right

Reply:
yep, LGTM. thanks!